lib/packagebuilder.go

```diff
@@ -84,6 +84,7 @@ var metapaths = []metapath{
 	metapath{path: "LeadConvertSettings", name: "LeadConvertSettings"},
 	metapath{path: "letterhead", name: "Letterhead"},
 	metapath{path: "matchingRules", name: "MatchingRules"},
```

Comment:
this file already has matchingRule path (line 86) but with wrong name (plural), should be singular. please remove line above

Reply:
Take a look at the [examples in the docs](https://developer.salesforce.com/docs/atlas.en-us.api_meta.meta/api_meta/meta_matchingrule.htm).  This metadata object is unusual in that it uses the plural name when fetching using a wildcard:

```
<?xml version="1.0" encoding="UTF-8"?>
<Package xmlns="http://soap.sforce.com/2006/04/metadata">
  <types>
    <members>*</members>
    <name>MatchingRules</name>
  </types>
  <version>42.0</version>
</Package>
```

Reply:
so, perhaps, you need to remove the next line (line 87)? 
```
metapath{path: "matchingRules", name: "MatchingRule"},
```

Reply:
I think they're both needed to support `force fetch -t MatchingRules` and `force fetch -t MatchingRule -n Account.AccountMatchingRule`.

Have you found any problems with having both?

Reply:
I just worried about duplication and incorrect package.xml generation for `force push -f ` `force push -t -n`

I did not test it yet, to be honest